lnwallet/interface_test.go

```diff
@@ -1474,106 +1531,213 @@ func TestLightningWallet(t *testing.T) {
 		aliceWalletController lnwallet.WalletController
 		bobWalletController   lnwallet.WalletController
 	)
-	for _, walletDriver := range lnwallet.RegisteredWallets() {
-		tempTestDirAlice, err := ioutil.TempDir("", "lnwallet")
-		if err != nil {
-			t.Fatalf("unable to create temp directory: %v", err)
-		}
-		defer os.RemoveAll(tempTestDirAlice)
 
-		tempTestDirBob, err := ioutil.TempDir("", "lnwallet")
-		if err != nil {
-			t.Fatalf("unable to create temp directory: %v", err)
-		}
-		defer os.RemoveAll(tempTestDirBob)
+	tempTestDirAlice, err := ioutil.TempDir("", "lnwallet")
+	if err != nil {
+		t.Fatalf("unable to create temp directory: %v", err)
+	}
+	defer os.RemoveAll(tempTestDirAlice)
+
+	tempTestDirBob, err := ioutil.TempDir("", "lnwallet")
+	if err != nil {
+		t.Fatalf("unable to create temp directory: %v", err)
+	}
+	defer os.RemoveAll(tempTestDirBob)
 
-		walletType := walletDriver.WalletType
-		switch walletType {
-		case "btcwallet":
-			aliceChainRPC, err := chain.NewRPCClient(netParams,
+	logger := btclog.NewBackend(os.Stdout)
```

Comment:
Loggin to `stdout` by default can be very verbose. Instead, we can guard this behind a command flag.